packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -3458,11 +3483,12 @@ function flushLayoutEffects(
   if (rootDidHavePassiveEffects) {
     // This commit has passive effects. Stash a reference to them. But don't
     // schedule a callback until after flushing layout work.
```

Comment:
I feel like we can remove this comment block now. We certainly don't stash a reference at this point anymore. And the scheduling is now obvious by advancing the phase.